cellpose/gui/guiparts.py

```diff
@@ -490,6 +491,22 @@ def update_minimap(self, parent):
         else:
             self.setFixedSize(400, 400)
 
+    def mousePressEvent(self, event):
+        """
+        Handles mouse press events on the minimap.
+        This method is triggered when the user clicks on the minimap,

```

Comment:
I would add that a left mouse click is required (since we also use the right click (for the adjustment in the former ticket 11))